src/python/WMCore/Agent/Daemon/Create.py

```diff
@@ -74,14 +75,14 @@ def daemonize(stdout='/dev/null', stderr=None, stdin='/dev/null',
     # Open file descriptors and print start message
     if not stderr:
         stderr = stdout
-    si = file(stdin, 'r')
-    so = file(stdout, 'a+')
-    se = file(stderr, 'a+', 0)
+    si = open(stdin, 'r')
```

Comment:
Hmm, has this been rebased on top of: https://github.com/dmwm/WMCore/pull/10057
instead of the PR you mention at the top level description?